### PR TITLE
Answer: 1

### PR DESCRIPTION
**Pull request by Danevandy99**
## Checklist for challenge submission

- [x] Start your PR message with Answer:${challenge_number}

### CODE REVIEW

File: apps/angular/projection/src/app/ui/list-item/list-item.component.ts [hunk 1]
```diff
@@ -1,35 +1,19 @@
...
-
-  delete(id: number) {
-    if (this.type === CardType.TEACHER) {
-      this.teacherStore.deleteOne(id);
-    } else if (this.type === CardType.STUDENT) {
-      this.studentStore.deleteOne(id);
-    }
-  }
+  id = input.required<number>();
+  onDelete = output<number>();
```
Comment (tomalaforge) [inline apps/angular/projection/src/app/ui/list-item/list-item.component.ts:18]: id is not needed, onDelete is just an event emitter

File: apps/angular/projection/src/app/ui/card/card.component.ts [hunk 2]
```diff
@@ -1,61 +1,49 @@
...
-      <img
-        *ngIf="type === CardType.TEACHER"
-        src="assets/img/teacher.png"
-        width="200px" />
-      <img
-        *ngIf="type === CardType.STUDENT"
-        src="assets/img/student.webp"
-        width="200px" />
+      [class]="customClass()">
+      <img headerImage [src]="headerImageUrl()" width="200px" />
```
Comment (tomalaforge) [inline apps/angular/projection/src/app/ui/card/card.component.ts:18]: you can use the image directive for better performance

**Comment by tomalaforge** [comment]
nice job

File: apps/angular/projection/src/app/ui/list-item/list-item.component.ts (hunk 1 above)
Comment (Danevandy99) [reply apps/angular/projection/src/app/ui/list-item/list-item.component.ts]: Thanks, should be removed now.

File: apps/angular/projection/src/app/ui/card/card.component.ts (hunk 2 above)
Comment (Danevandy99) [reply apps/angular/projection/src/app/ui/card/card.component.ts]: I added the new NgOptimizedImage directive.

**Pull request closed without merging**